tests/test_types_itp_lithologischebeschrijvingen.py

```diff
@@ -0,0 +1,123 @@
+"""Module grouping tests for the
+pydov.types.interpretaties.InformeleStratigrafie class."""
```

Comment:
This should be LithologischeBeschrijvingen

Reply:
@Roel:
- I will have a look at the code to remove CPT from pkeys
- concerning the Notebook: does it keep the downloaded xmls in memory?

Reply:
I don't think the xml's are kept in memory, but maybe the resulting dataframe is?